templates/server.yaml

```diff
@@ -13,7 +13,6 @@ spec:
         {{- include "kepler.selectorLabels" . | nindent 8 }}
     spec:
       hostNetwork: true
-      serviceAccountName: {{ .Values.serviceAccount.name }}
```

Comment:
please keep this line

Reply:
Sure. I'm sorry, I may have deleted it accidentally.